NoneType has no Session()

The merge of #105 seems to have broken unit tests (and the client ^_^)

```
>           self.connection = requests.Session()
E           AttributeError: 'NoneType' object has no attribute 'Session'

../official_mwclient/mwclient/mwclient/client.py:91: AttributeError
============================================================================ 20 failed, 21 passed in 1.24 seconds ============================================================================
```

I tried with the python interprer, I have the same results:
```
>>> import mwclient
>>> mwclient.__ver__
'0.8.2.dev1'
>>> mwclient.Site('commons.wikimedia.org')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "mwclient/client.py", line 91, in __init__
    self.connection = requests.Session()
AttributeError: 'NoneType' object has no attribute 'Session'
```

I'd say masking the requests package with an argument name is not a good idea ^_^

--- a/mwclient/client.py
+++ b/mwclient/client.py
@@ -55,7 +55,7 @@ class Site(object):
     def __init__(self, host, path='/w/', ext='.php', pool=None, retry_timeout=30,
                  max_retries=25, wait_callback=lambda *x: None, clients_useragent=None,
                  max_lag=3, compress=True, force_login=True, do_init=True, httpauth=None,
-                 requests=None):
+                 reqs=None):
         # Setup member variables
         self.host = host
         self.path = path
@@ -64,7 +64,7 @@ class Site(object):
         self.compress = compress
         self.max_lag = text_type(max_lag)
         self.force_login = force_login
-        self.requests = requests or {}
+        self.requests = reqs or {}
 
         if isinstance(httpauth, (list, tuple)):
             self.httpauth = HTTPBasicAuth(*httpauth)

